spec/plugin_spec_helper.rb

```diff
@@ -96,16 +96,16 @@ def mocks_for_creating role_class
       end
 
       def disable_flash_sweep
-       @controller.instance_eval{allow(flash).to receive(:sweep)}
+        @controller.instance_eval { allow(flash).to receive(:sweep) }
       end
 
       def disable_log_requesting_user
         allow(@controller).to receive(:log_requesting_user)
       end
 
-      def response_should_render method, *params
+      def response_should_render(method, *params)
         unless @page
-          @page ||= double("page")
+          @page ||= double('page')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.